### DISCUSSION TITLE
Add Create Folder ability to Image picker

### DISCUSSION THREAD

**Discussion by X3Technology** [vanjarosoftware/Vanjaro.Platform#1699] [General]
I keep having to go the Asset manager to add a new folder.  It would be nice if I could do it directly from the Image picker.

**Comment by meetmandeep** [comment]
I agree. I don't see why the same functionality set cannot be available in image/video picker. The asset manager dynamically adjusts based on the logged in user permission. 

It's possible we have an issue created on this already; if not, we'll move this to issues.
